### PR TITLE
Feature/admins-unification

### PR DESCRIPTION
**Pull request by yurii-krupa**
Code update to work with common data/network manager from pull request #18.
For workability have to aprove request #18 .

### CODE REVIEW

**Comment by yurii-krupa** [comment]
[Admins](https://trello.com/c/l8jnvKpM/5-admins)

**Comment by iostaf** [comment]
I'll review this request when #18 is merged.

File: DTAdmin/Admins/Admin.storyboard [hunk 1]
```diff
@@ -45,10 +45,13 @@
                                     <outlet property="delegate" destination="fkA-hd-7Bn" id="vrs-2P-Gbf"/>
                                 </connections>
                             </tableView>
-                            <searchBar contentMode="redraw" fixedFrame="YES" translatesAutoresizingMaskIntoConstraints="NO" id="cgr-r7-DQU">
+                            <searchBar contentMode="redraw" fixedFrame="YES" placeholder="Search by username albo email" translatesAutoresizingMaskIntoConstraints="NO" id="cgr-r7-DQU">
```
Comment (iostaf) [inline DTAdmin/Admins/Admin.storyboard:48]: Typo in `albo`

**Review by iostaf — APPROVED** [review]

File: DTAdmin/Admins/Admin.storyboard (hunk 1 above)
Comment (yurii-krupa) [reply DTAdmin/Admins/Admin.storyboard:48]: In Polish language "albo" means "or".
I forget to correct it, fix in next PR!

**Pull request merged**